src/legacy/server/sass/build.js

```diff
@@ -91,8 +91,7 @@ export class Build {
       sourceMap: true,
       sourceMapEmbed: true,
       includePaths: [
-        resolve(__dirname, '../..'),
```

Comment:
This is really the change in this PR that isn't strictly just adjusting paths. The existing sass implementation for plugins allowed importing from anywhere inside `src`. I spoke to @tylersmalley about this and we agreed to just let plugin sass import relative to root rather than update this to import from within `src/legacy`.

There were a couple reasons for this:

1. We can't statically analyze sass, so any sort of bulk updates to paths will need to be done through find/replace. Something more specific like `src/legacy/ui` is much easier to grep for than just `ui`.
2. If you don't know exactly where to look, finding this path configuration is challenging since it's just string-based relative paths.
3. Why not treat paths a little less magically?

We both also agreed that node_modules should probably be removed as well so folks would need to import those relative root as well, but that felt out of scope of this PR since it wasn't already touching any of those imports.